### PR TITLE
feat: add homepage and auth pages, jsx only

### LINKED ISSUE #2: add homepage

### PR DESCRIPTION
**Pull request by Bhakti199**
Here I added jsx for homepage , login and signup. Tried using light colors for aesthetic experience.  #2

### CODE REVIEW

**Review by mariyasada — APPROVED** [review]
UI looks cool .good to go

File: src/Pages/AuthPages/SignUpPage.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import { Link } from "react-router-dom";
+
+export const SignUpPage = () => {
+  return (
+    <div className="auth-page">
+      <form
+        className="login-form"
+        onSubmit={(event) => {
+          loginDetailsHandler(event);
+        }}
```
Comment (JantuDeb) [inline src/Pages/AuthPages/SignUpPage.jsx:11]: onSubmit={loginDetailsHandler}

**Review by JantuDeb — APPROVED** [review]
Overall it is looking good

File: src/Pages/AuthPages/SignUpPage.jsx (hunk 1 above)
Comment (Bhakti199) [reply src/Pages/AuthPages/SignUpPage.jsx:11]: I will remove it.

**Pull request merged**